lib/oli_web/live/progress/student_resource_view.ex

```diff
@@ -57,8 +57,14 @@ defmodule OliWeb.Progress.StudentResourceView do
 
             changeset =
               case resource_access do
-                nil -> nil
-                _ -> ResourceAccess.changeset(resource_access, %{})
+                nil ->
+                  nil
+
+                _ ->
+                  ResourceAccess.changeset(resource_access, %{
+                    # limit score decimals to two significant figures, rounding up
+                    score: Float.round(resource_access.score, 2)
```

Comment:
Do we actually want to round the score in the database as well? I thought we wanted to store it as is, and then format the value only when it's displayed.